src/operator.cpp

```diff
@@ -151,7 +151,6 @@ void Operator::createCluster(QStringList args)
 
 void Operator::startCommandStarting()
 {
-    commandStarting();
```

Comment:
This doesn't fix it for me, for me, on main or this PR I get the same result. The start window shows up on the first start, but any subsequent starts the loading window doesn't show up.

Reply:
Fixed with https://github.com/kubernetes-sigs/minikube-gui/pull/86, could you revert removing this line and then it'll be good to merge